types/network/modules/hammer.d.ts

```diff
@@ -0,0 +1,2 @@
+declare const whoKnowsWhat: any
```

Comment:
can't we use somethink like `@types/hammerjs`?

Reply:
Good idea, I'll look into it.

Reply:
It was not imported from anywhere :smile: so I deleted it.